lib/cequel/record/belongs_to_association.rb

```diff
@@ -17,6 +17,8 @@ class BelongsToAssociation
       attr_reader :name
       # @return [String] name of parent class
       attr_reader :association_class_name
+      # @return [String] name of parent class
+      attr_reader :foreign_key
```

Comment:
Given that there can be multiple foreign key fields I think this option and attr should be plural.